types/office-js-preview/index.d.ts

```diff
@@ -509,6 +509,32 @@ declare namespace Office {
          */
         Text,
     }
+    /**
+     * Specifies the device capability to which an add-in is requesting access.
+     *
+     * @remarks
+     *
+     * **Applications**: Excel on the web, Outlook on the web, PowerPoint on the web, and Word on the web running in Chromium-based browsers,
+     * such as Microsoft Edge and Google Chrome.
+     *
+     * **Requirement set**: {@link https://learn.microsoft.com/javascript/api/requirement-sets/common/device-permission-service-requirement-sets | DevicePermission 1.1}
+     */
+    enum DevicePermissionType {
+        /**
+         * The add-in is requesting access to the user's camera.
+         */
+        camera,
+        /**
+         * The add-in is requesting access to the user's geolocation.
+         *
+         * **Important**: Access to a user's geolocation is only supported in add-ins running in Outlook on the web.
```

Comment:
I think the add-in's context here is clear.
```suggestion
         * **Important**: Access to a user's geolocation is only supported in Outlook on the web.
```